docker-entrypoint.sh

```
@@ -0,0 +1,35 @@
+    EXTRA_ARGS="$EXTRA_ARGS --enable-multi-user true"
+fi
+
+if [ "${ENABLE_MCP}" = "true" ]; then
+    EXTRA_ARGS="$EXTRA_ARGS --enable-mcp true --mcp-port ${MCP_PORT:-3001}"
+fi
+
+exec ad4m-executor run \
+    --app-data-path /data \
+    --localhost false \
+    --run-dapp-server true \
+    $EXTRA_ARGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)